js/src/Ice/TimerUtil.browser.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) ZeroC, Inc.
+
+let Timer = {};
+
```

Comment:
This is the browser code, which was previously in TimerUtil.js, with and if/else for Node.js/Browser versions.